pyproject.toml

```diff
@@ -12,7 +12,7 @@ packages = [{include = "django_api_decorator"}]
 [tool.poetry.dependencies]
 python = "^3.10"
 Django = ">=3"
-pydantic = "^2.0"
+pydantic = "^2.4.2"
```

Comment:
Do we need to update this? Might be better to just update the lock file, otherwise we force downstream projects to also upgrade

Reply:
Ops, no not needed ⭐ 
Forgot to close this. Thanks!